src/lib/gadgets/range-check.ts

```diff
@@ -3,6 +3,10 @@ import * as Gates from '../gates.js';
 
 export { rangeCheck64 };
 
+/**
```

Comment:
I'm assuming that this function will be what users will see for the exported comments, I would love to see a more helpful doc comment here, do you think something like this would be okay?

```ts
/**
 * Validates if the given field value fits within the range [0, 2^64).
 * 
 * This function checks if the provided field value can be represented within 64 bits.
 * If the field value exceeds 64 bits, an error is thrown.
 *
 * @param x - The field value to be checked.
 * 
 * @throws Will throw an error if the field value exceeds 64 bits.
 * 
 * @example
 * ```ts
 * const field = new Field(12345678n);
 * rangeCheck64(field);  // Validates without throwing an error
 * 
 * const invalidField = new Field(12345678901234567890123456789012345678n);
 * rangeCheck64(invalidField);  // Throws an error since it exceeds 64 bits
 * ```
 */
 ```

Reply:
Or maybe if we have another import that's used, we can mention the TSDoc there too?